kernel/fs_node/src/lib.rs

```diff
@@ -190,4 +197,15 @@ impl FileOrDir {
             FileOrDir::Dir(_) => true,
         }
     }
+
+    /// Returns the size of the file in bytes, or directory disk space.
+    pub fn get_file_size(file_or_dir: &FileOrDir) -> usize {
```

Comment:
This is redundant, see the existing `len()` method for files (part of the `KnownLength` trait). See my other comment about handling sizes for directories.